src/Compilers/CSharp/Portable/Binder/Binder_Conversions.cs

```diff
@@ -394,7 +394,7 @@ private BoundExpression CreateStackAllocConversion(SyntaxNode syntax, BoundExpre
                     break;
                 case ConversionKind.StackAllocToSpanType:
                     CheckFeatureAvailability(syntax, MessageID.IDS_FeatureRefStructs, diagnostics);
-                    stackAllocType = Compilation.GetWellKnownType(WellKnownType.System_Span_T).Construct(elementType);
+                    stackAllocType = Compilation.GetWellKnownType(WellKnownType.System_Span_T, recordUsage: false).Construct(elementType);
```

Comment:
>recordUsage: false [](start = 95, length = 18)

Why not record usage here? This seems like it will definitely be used by the final assembly?

Reply:
> Why not record usage here? This seems like it will definitely be used by the final assembly?

An attempt to classify conversion is not a usage. It can be made by any external consumer through public API.

---
In reply to: [351355183](https://github.com/dotnet/roslyn/pull/40010#discussion_r351355183) [](ancestors = 351355183)